packages/utils/src/ReducerRegistry/ReducerRegistry.ts

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import { applyMiddleware, combineReducers, createStore, compose } from 'redux';
 
-export function applyReducerHash(reducerHash, initialState = {}) {
-  return function (state = initialState, action) {
+export function applyReducerHash(reducerHash: any, initialState: Record<string, unknown>) {
```

Comment:
You have to initialize the `initialState` argument. Or will be treated as a required attribute.

Reply:
I am a bit confused regarding this, would adding something like `initialState?: Record<string, unknown>` be fine in this case? 😶

Reply:
In the original code there is a default value: `initialState = {}`, you miss it the ts version.